src/k8smanager.ts

```diff
@@ -224,10 +224,14 @@ export class K8sManager implements IK8sManager {
                     promises.push(this._api.createNamespacedReplicationController(this._namespace, doc));
                     break;
                 case 'Deployment':
-                    const imageName: string = doc.spec.template.spec.containers[0].image;
+                    let imageName: string = doc.spec.template.spec.containers[0].image;
                     if (imageName.includes('{runtime}')) {
                         doc.spec.template.spec.containers[0].image = imageName.replace('{runtime}', this._config.Runtime);
                     }
+                    imageName = doc.spec.template.spec.containers[0].image;
```

Comment:
This is the same as in line 227 -
imageName = doc.spec.template.spec.containers[0].image;

remove ?

Reply:
Yes it is expected. Basically i need to reassign with the changed value online 229